### DISCUSSION TITLE
How do I use multiple Kubernetes clusters?

### DISCUSSION THREAD

**Discussion by SandraGH5** [flyteorg/flyte#1074] [Q&A] [answered]

**Comment by SandraGH5** [comment, marked as answer]
Please visit "Using Multiple Kubernetes Clusters" at https://docs.flyte.org/projects/cookbook/en/latest/auto/deployment/workflow/multiple_k8s.html.

**Comment by kumare3** [comment]
Also, feel free to ping the Slack channel for additional help in setting up multi-k8s environment
